hail/python/hailtop/hailctl/dataproc/cli.py

```diff
@@ -95,5 +99,8 @@ def main(args):
         'describe': describe,
     }
 
-    args, pass_through_args = parser().parse_known_args(args=args)
+    args, pass_through_args = p.parse_known_args(args=args)
+    if "module" not in args:
+        p.error('positional argument required')
```

Comment:
why is this newly necessary? Would I not get the help message if I run `hailctl --beta`?

Reply:
Yes